make-it/EmMakefile

```diff
@@ -10,6 +10,7 @@ SITEDIR=$(VIREO_SDK_DIR)/Documents/gh-pages/playground
 TARGETDIR=$(VIREO_SDK_DIR)/dist
 CORESOURCEDIR=$(VIREO_SDK_DIR)/source/core
 IOSOURCEDIR=$(VIREO_SDK_DIR)/source/io
+UTSOURCEDIR=$(VIREO_SDK_DIR)/source/unittest
```

Comment:
Actually, the body of the cpp files are commented out #if VIREO_UNIT_TEST is not 1.
The only code that's always compiled in is the code that conditionally runs the test list if it's not empty, which is pretty small.